src/components/PeopleFilters.tsx

```diff
@@ -2,22 +1,115 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+type Props = {
+  sex: string;
+  centuries: string[];
+};
+
+export const PeopleFilters: React.FC<Props> = ({ sex, centuries }) => {
+  const [searchParam, setSearchParam] = useSearchParams();
+  const defaultNameValue = searchParam.get('name') || '';
+
+  const sexActive = ({ isActive, type }: { isActive: string; type: string }) =>
+    classNames({
+      'is-active': isActive === type,
+    });
+
+  const allClass = classNames({
+    'is-outlined': centuries.length > 0, // Add 'is-outlined' if any centuries are selected
+  });
+
+  const centuryButtonClass = (value: string) =>
+    classNames('button mr-1', {
+      'is-info': centuries.includes(value), // Add 'is-info' if century is selected
+    });
+
+  const handleSexInput = (type: string) => {
+    const params = new URLSearchParams(searchParam);
+
+    if (type) {
+      params.set('sex', type);
+    } else {
+      // Remove the 'sex' parameter if 'All' is selected
+      params.delete('sex');
+    }
```

Comment:
In the `handleSexInput` function, ensure that the `type` parameter is correctly handled. If `type` is an empty string, the 'sex' parameter should be removed, which is correctly implemented. However, make sure that the logic for setting the 'sex' parameter is consistent with the expected values ('m', 'f', or '').